### PR TITLE
[Spanish] Create strings.xml

### PR DESCRIPTION
**Pull request by DD21S**
Addition of the Spanish (US) language.

### CODE REVIEW

**Comment by z-huang** [comment]
Is this and #119 the same? Is `es` equal with `es-US`?

**Comment by DD21S** [comment]
> Is this and #119 the same? Is `es` equal to `es-US`?

Yes, they're the same.

File: app/src/main/res/values-es-rUS/strings.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    <string name="sort_by_artist">Artista</string>
+    <string name="sort_by_create_date">Fecha de incorporación</string>
+
+    <!-- Snackbar -->
+    <plurals name="snack_bar_delete_song">
+        <item quantity="one">%d canción fue eliminada.</item>
+        <item quantity="other">%d canciones fueron eliminadas.</item>
+    </plurals>
+    <plurals name="n_selected">
+        <item quantity="other">%d seleccionadas</item>
```
Comment (z-huang) [inline app/src/main/res/values-es-rUS/strings.xml:132]: For locale "es" (Spanish) the following quantities should also be defined: one

**Review by z-huang — CHANGES_REQUESTED** [review]
One line needs to be added.

**Comment by z-huang** [comment]
> Yes, they're the same.

Then I'll merge the one that passes the check first.

**Review by z-huang — APPROVED** [review]

**Comment by z-huang** [comment]
Thanks!

**Pull request merged**